src/commands/lambda/instrument.ts

```diff
@@ -207,6 +207,20 @@ export class InstrumentCommand extends Command {
     }
 
     const configList = configGroups.map((group) => group.configs).reduce((a, b) => a.concat(b))
+    // Returns false if not all runtimes are of the same RuntimeType across multiple functions
+    const runtimesAreUniform = configList
```

Comment:
I would break this out into its own little function and write unit tests for it.